qemu/tests/thin_provisioning.py

```diff
@@ -0,0 +1,170 @@
+import re
+import os
+import logging
+
+from autotest.client.shared import error
+from autotest.client.shared import utils
+from autotest.client import os_dep
+from virttest import env_process
+from avocado.utils import process
+from avocado.core import exceptions
+
+
+def get_scsi_disk(session=None):
```

Comment:
Shall we get all this function get_scsi_disk(), get_provisioning_mode(), get_allocation_bitmap(), get_guest_discard_disk() inside a class? or add the function to avocado frame? As I found them also implemented and used in block_discard.py. So we can use it directly next time without any duplicate code introduced.